Dockerfile

```diff
@@ -1,12 +1,18 @@
-FROM gcc:latest
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
```

Comment:
Please do not modify upstream Dockerfiles. I recommend restoring the original and creating a separate Dockerfile for fuzz testing (i.e. Dockerfile.mayhem).